docs/plugins/virtual_environment_docs/2-deploying-your-plugin.md

```diff
@@ -0,0 +1,109 @@
+# Deploying your plugin  
+
+This guide explains some of the techniques you can use to deploy your plugin. 
+
+## This guide covers:   
+* [PyPI and Anaconda](#pypi-and-anaconda)
+    - [Package formats (PyPI)](#package-formats-pypi)
+    - [PyPI](#pypi)
+* [Deploying plugins](#deploying-plugins)
+    - [Automatically via GitHub actions](#automatically-via-github-actions)
+    - [Anaconda cloud via conda-forge](#anaconda-cloud-via-conda-forge)
+
+
+## PyPI and Anaconda
+PyPI and Anaconda are two options for how you distribute your package and allow your users to more easily find and install it.  Try to deploy to both! But for now, try to at least use PyPI. You can always also provide your users with manual installation instructions (e.g. if you want them to use `conda` or have specific dependencies).
+
+### Package formats (PyPI)
+**sdist** means source distribution. An **sdist** includes all of the files that are required to *build* your package. An **sdist** may require specific additional software (e.g. compilers) to actually build.
+
+**wheel** is a prebuilt package, ready to drop into site-packages. It includes compiled OS-specific extensions (if applicable).
+
+You are *strongly* encouraged to ship both! If the wheel is not present, pip will try to build it from the sdist before installation, and that may fail depending on the package. To see if a given package ships a wheel, check here: https://pypi.org/project/napari/#files
+
+**Note:** This goes for dependencies too! Check all your dependencies for wheel availability.  
+
+### PyPI
+**[build](https://pypa-build.readthedocs.io/en/latest/ )** : package builder that bundles your source code into `sdist` or `wheel` distributions
+
```

Comment:
And the code block for the comment above...:

```console
pip install build
python -m build .
```

Reply:
I will add this.

Reply:
Done.